src/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+set(
+    sources
+    rdaddr.c
+    rdavl.c
+    rdcrc32.c
+    rdkafka.c
+    rdkafka_assignor.c
+    rdkafka_broker.c
+    rdkafka_buf.c
+    rdkafka_cgrp.c
+    rdkafka_conf.c
+    rdkafka_event.c
+    rdkafka_feature.c
+    rdkafka_metadata.c
+    rdkafka_metadata_cache.c
+    rdkafka_msg.c
+    rdkafka_offset.c
+    rdkafka_op.c
+    rdkafka_partition.c
+    rdkafka_pattern.c
+    rdkafka_queue.c
+    rdkafka_range_assignor.c
+    rdkafka_request.c
+    rdkafka_roundrobin_assignor.c
+    rdkafka_subscription.c
+    rdkafka_timer.c
+    rdkafka_topic.c
+    rdkafka_transport.c
+    rdlist.c
+    rdlog.c
+    rdports.c
+    rdrand.c
+    rdregex.c
+    rdstring.c
+    snappy.c
+    tinycthread.c
+)
+
+if(WITH_SASL)
+  list(
+      APPEND
+      sources
+      rdkafka_sasl.c
+      rdkafka_sasl.h
+      rdkafka_sasl_cyrus.c
+      rdkafka_sasl_int.h
+  )
+  if(WIN32)
+    list(APPEND sources rdkafka_sasl_win32.c)
+  endif()
+endif()
+
+if(WITH_ZLIB)
+  list(APPEND sources rdgz.c)
+endif()
+
+if(NOT HAVE_REGEX)
+  list(APPEND sources regexp.c)
+endif()
+
+add_library(rdkafka ${sources})
+
+# Support '#include <rdkafka.h>'
```

Comment:
This is fix for this file: https://github.com/edenhill/librdkafka/blob/89ed67fabb3da66b6c0ecf7fd704c0e1381348fa/examples/rdkafka_performance.c#L43